src/components/SQFormDialogStepper/SQFormDialogStepper.tsx

```diff
@@ -84,17 +136,17 @@ export function SQFormDialogStepper({
   title,
   enableReinitialize = false,
   muiGridProps = {},
-  dialogProps = {},
+  dialogProps = {open: isOpen},
```

Comment:
What was the need for this change?

Reply:
I see why you decided to make this change. I have an issue with it, though. You removed passing `open` to the component below. What this means is that if the consumer of this component passes in a value for `dialogProps` it'll overwrite the default and as such they won't have `open`. However, TS won't let them do that if TS is being used to consume this library. So it'd be confusing for the user to pass `open` as a prop and in this dialogProps prop.

If the consumer is using JS alone, no TS, then they can pass whatever they like here and they might not get `open`.

With that being said, I think we should remove the default from this prop, pass `isOpen` to the component that needs it directly. And do a check before spread `dialogProps`. Say `<Component ...(dialogProps || {})>`

Reply:
Makes sense @20BBrown14.  Will change this. Thanks.

Reply:
I guess we won't be able to use `DialogProps` then to type the dialogProps since it requires the `open` property.